telegram/ext/updater.py

```diff
@@ -485,6 +492,8 @@ def _join_threads(self):
 
     def signal_handler(self, signum, frame):
         self.is_idle = False
+        if self.persistence:
```

Comment:
why?

Reply:
Cause otherwise we impede the ability to exit immediately? Like if the user presses CTRL-C (or whatever) twice, it shouldn't save persistence twice, it just quit ASAP imo

Reply:
Good point